### DISCUSSION TITLE
Medigel and Biofoam: Determine how they fit

### DISCUSSION THREAD

**Discussion by WitchFreya** [WitchFreya/SWSAddons#51] [Discovery]
This is something to think about for a bit as well. Medigel and biofoam are both signature parts of the setting and it would be neat to provide players the ability to use them in game, but I'd like to figure out what niche they might fill first.

It might potentially be nice for these to be _the_ items that everyone but medics carry; possibly some form of integrated painkiller as well. Things to think about.

**Comment by EidolonForgery** [comment]
I might rewrite them to be a combo quick-clot and tiny painkiller? That'd fulfill a _lot_ of needs. Alternately, they could be a single-use stitch kit with built-in painkillers that don't lower heartrate - it's meant to be an analgesic foam in there.

**Comment by WitchFreya** [comment]
Not a bad idea, I'd be interested to see how it plays out at least.

Relevant file on ACE's end as a reference point: https://github.com/acemod/ACE3/blob/bed8844b7451c5d4dca89daa41cb4f4add09cb44/addons/medical_treatment/ACE_Medical_Treatment.hpp

**Comment by swiftysmoon** [comment]
I'm curious to see how it plays out, but I'm admittedly a bit mixed. I like that idea of combining something with a pain killer, because it keeps the morphine in the hands of the cls/medic, but here are some of my disjointed thoughts. 

Quickclot+analgesic:
I like that this allows players a little bit of discretion in when they choose to use their supplementary medical supplies that ideally also gets them to consider _how_ they're using them. If in order to use that pain killer they have to also use the equivalent of a quickclot, they're hopefully led to make smarter choices about how to do that with a finite number of supplies, and determine when it might really be more prudent to let the cls work their magic. 

From a combat medic specific standpoint, it would be really nice to be able to instruct someone to biofoam themselves and pull overwatch in a mass cas and know that a single, quick action has given them twice the stability. 

That said, I do worry about a couple things. I wonder how it might effect mas/high casualty situations if players stop carrying bandages. Because there are definitely some times where being able to instruct players to bandage strategically has been the make or break part of recovering multiple casualties for me. 

I could also see the potential for folks being more reluctant/unhappy to use their supplies during a mass cas at the medics behest because they're spending two resources in one and those situations can be pretty resource heavy. 

Stitch kit + analgesic:
You've probably considered this, but I would caution against allowing people to carry too many of these. Resource scarcity might help avoid some of the problems that packing the analgesic into a baby surgical kit might cause (over-reliance at the expense of the cls role, mainly). Also, I selfishly like that stitching is cls specific, and if folks are allowed to take too many it effectively removes the task from the CLS's hands. My concern, specially, is that folks might use them in less intense combat situations or because they're only a little injured and want to avoid the time sink of needing paks/medical attention in order to stay in the action/because they don't want to bother the cls with small things. Those of us that play cls/medic because we like the role might find that frustrating if it became the norm, on a slow ops especially. 

That said, I do prefer this idea to packaging an analgesic with the bandage function because in my weird little brain it makes more sense for medigel to work as a stitch+pain killer and biofoam to be a heavier bandage replacement so folks are limited to an amount that makes sense for general play. 

I dont want it to be harder for folks to do their own medical, but I do like the idea of giving them a practical reason to reconsider the necessity of doing it themselves when their CLS is free, and I think switching non medic roles over to biofoam/medigel presents a good opportunity to moderate that naturally. I would also, personally, prefer that whatever we pack that analgesic into doesn't lower heart rate, because people are going to use the supplies you give them, and they aren't always going to communicate that they have. I'm sure yall have thought of this, too, but I really don't want the actual hell of having to determine if patients have  lowered their own heart rate too much during high stress medical situations.

**Comment by EidolonForgery** [comment]
As an afterthought, it'd be nice to have one of the two things just be a better quickclot. Like the wound reopening chance is lower or it applies faster, but takes up a decent amount of space.

**Comment by WitchFreya** [comment]
@swiftysmoon Added you as a collaborator, just as a head's up!

Additionally, fwiw your feedback on this is what I prioritize most highly—I'm considering cutting a development branch and sending you the dev version when it's ready so that we can test things out together.

As for ideas; I was discussing a few things with Jazz at coffee this morning and the concepts we arrived at were the following:

## Medigel

- **Primarily Carried By**: General roles other than CLS/Medic.
- **Application Time**: 2-4 seconds or so.
- **Duration**: 1-2 minutes.
- **Function**: Emergency pain-killer. Short-release and short-lived; reduce pain amount and pain incoming. Doesn't lower heartrate, does increase blood pressure (think like, 250-500ml IV). Blood pressure increase will linger post-expiration.

The motivation would be something that a rifleman or similar could apply really quickly (potentially even on a keybinding) that would keep them going for longer during a combat and reduce the need for CLS to PAK someone just because their pain is super high.

## Biofoam

- **Primarily Carried By**: CLS/Medic
- **Application Time**: 5-10 seconds or so.
- **Function**: Effectively a heavier but bulk elastic that is quick to apply.

The idea being for situations where a medic is dealing with a mass cas or someone at fatal and they can hit a wounded area with this to immediately close up most if not all injuries on that body part. My thought is that, as we start dealing with Covenant and such, we're going to have more situations that are more dangerous and having medics have an "Oh fuck" button seems like it could be handy.

---

Let me know how that looks to y'all!

**Comment by swiftysmoon** [comment]
@WitchFreya Oooh, this. I like this. It potentially mitigates a couple of problems I'd been knocking around in my head, (both how to efficiently administer/monitor pain medication when the most useful application is usually during a high intensity situation, and how we were going to handle mas cas when a single barage from a banshee can take an entire fireteam to fatal.) I'm sure we'll find some tweaking to do in practice, but conceptually this gets the green light from me. 

And I really appreciate that Freya! I'm glad for the chance to help out, and I'd absolutely love to dig my hands into the testing phase with you. :3

**Comment by WitchFreya** [comment]
Closing out the discussion as I've opened a ticket for it #73.

Adjusted the AC for Medigel to be clearing trauma on bruises or burns as an alternative to PAK for minor injuries, once the PR is open we can look it over a bit and see how we feel.
